doc/design.md

```diff
@@ -755,37 +759,38 @@ d.insert<Data::Variance>("sample", dims);
 d.insert<Data::Value>("background", dims);
 d.insert<Data::Variance>("background", dims);
 
-// Add monitors as a nested dataset.
+// Add monitors as a nested dataset. In practice storing monitors as attribute
```

Comment:
This has confused me a little. I'm not seeing how the nesting actually happens. There is no additional dataset initialized for the monitors, and I don't believe that any of the `insert` overloads create one internally either. Would appreciate some clarification on this.

Reply:
`d.insert<Coord::Monitor>({Dim::Monitor, 2});` creates a variable for the monitors. `Coord::Monitor` defines the item type, in this case `Dataset`. No explicit data is given, so they are default-initialized.